CommonData/column.h

```diff
@@ -162,6 +162,8 @@ class Column : public DataSetBaseNode
 
 			void					checkForLoopInDependencies(std::string code);
 			const	stringset	 &	dependsOnColumns(bool refresh = true);
+			Json::Value				serialize()																const;
```

Comment:
Would it make sense to make `serialize()` and `deserialize()` virtual functions on `DataSetBaseNode`?